simplq/src/components/pages/Home/StaticInfos.jsx

```diff
@@ -127,11 +127,51 @@ export const ExtraInfo = () => (
   </div>
 );
 
-export const GetStarted = () => (
+export const Pricing = () => (
   <div data-aos="fade-up" className={`${styles['section']} ${styles['get-started']}`}>
-    <h2 data-aos="zoom-in">Ready to get started with SimplQ?</h2>
-    <div className={`${styles['button-group']}`}>
-      <StandardButton onClick={smoothScrollToHomePageTop}>Start Now</StandardButton>
+    <h2 data-aos="zoom-in">Pricing - Plans for every usecase</h2>
+    <div className={styles['container']}>
+      <div className={styles['price-card']}>
+        <h1>Free</h1>
+        <p className={styles['subtitle']}>Basic for small businesses.</p>
+        <li>
+          <ul>Unlimited Queues</ul>
+          <ul>20,000 tokens per month.</ul>
+        </li>
+        <p className={styles['amount']}>$0</p>
+        <StandardButton onClick={smoothScrollToHomePageTop}>Start for free</StandardButton>
+      </div>
+      <div className={styles['price-card']}>
+        <h1>Business</h1>
+        <p className={styles['subtitle']}>For bigger companies.</p>
+        <li>
+          <ul>Everything in the free plan.</ul>
+          <ul>Unlimited tokens per month.</ul>
+          <ul>Custom subdomains.</ul>
+          <ul>Multi-user support.</ul>
+          <ul>Priority Customer Support.</ul>
+        </li>
+        <p className={styles['amount']}>$8.99/mo</p>
+        <StandardButton onClick={smoothScrollToHomePageTop}>Upgrade</StandardButton>
+      </div>
+      <div className={styles['price-card']}>
+        <h1>Enterprise</h1>
+        <p className={styles['subtitle']}>Best for unique requirements that need to scale.</p>
+        <li>
+          <ul>Everything in the business plan.</ul>
+          <ul>No token or user limits.</ul>
```

Comment:
Only this point mentions users. I feel that we should either remove users here or add users to the other sections too.

Reply:
I'll remove this point, it's not needed.